bk_ssm_secrets/tests/test_helpers.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from . import helpers
+
+class TestHelpers(unittest.TestCase):
+    def test_url_to_slug(self):
+        assert(helpers.url_to_slug("https://github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("https://github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("git@github.com:a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("user@github.com:a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("https://git@github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("ssh://git@github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("ssh://git@github.com:22/a/b.git") == "github.com-22_a_b.git")
+        assert(helpers.url_to_slug("https://github.com:8080/a/b.git") == "github.com-8080_a_b.git")
```

Comment:
thanks @vgrigoruk ,
Does this handle URI's with the form: `https://github.com:8080/~a/b.git` etc ?
The `~` was replaced with `_` originally.

Reply:
Pushed a commit that fixes this :)